Sketcher/SketchUnmap.FCMacro

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+#####################################
+# Copyright (c) openBrain 2019
+# Licensed under LGPL v2
+#
+# This macro will unmap a sketch from eg. a face and make its placement absolute in the body. It proposes 3 options (forcely the first if sketch not in a body) :
+# * "Raw" mode => the sketch placement is made absolute in the body referential, nothing more
+# * "DP@Face mode" => a datum plane is created where the mapping face is, then the sketch is attached to it respecting its attachment offset
+# * "DP@Sketch" mode => a datum place is created where the sketch is (including attachment offset), then the sketch is attached to its origin
+#
+# Main use of the macro is to make design more robust to topological naming issue
+#
+# Version history :
+# *0.5 : beta release
+# *0.6 : some typo improvement + commenting for official PR
+#
+#####################################
+
+    __Name__ = 'SketchUnmap'
+    __Comment__ = 'Unmap a sketch & makes its placement absolute"'
+    __Author__ = 'openBrain'
+    __Version__ = '0.6'
+    __Date__ = '2019-06-10'
+    __License__ = 'LGPL v2'
+    __Web__ = 'https://www.freecadweb.org/wiki/Macro_SketchUnmap'
+    __Wiki__ = 'https://www.freecadweb.org/wiki/Macro_SketchUnmap'
+    __Icon__ = ''
+    __Help__ = 'Select the sketch to unmap (eg. in the tree view) then run the macro'
+    __Status__ = 'Beta'
+    __Requires__ = 'FreeCAD >= 0.17'
+    __Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=22&t=36078'
+    __Files__ = ''
```

Comment:
Files associated with this macro need to be added here.

Reply:
I don't have any ATM. I kept property for potential future icon. ;) Do you prefer I remove the line while there is no linked file ?

Reply:
@0penBrain ah, apologies, you were aware of this and you raise a good question.   
@galou what do think?

Reply:
@luzpaz no problem. :) Actually that was a guess. There is also the __Icon__ metadata that I kept blank. I can't determine from the repo readme if icon file shall also be included in the __Files__ metadata ??? So I kept it blank to think about it. I commented out this line so everything is clear (hopefully). :)

Reply:
The `__Files__` metada should exits even if  it's empty. We actually do not do anything right now with the `__Icon__` metadata, as far as I know.

Reply:
@galou thanks for clarifying. Could this be indicated in the README.md of the repo with something like "Please keep all metadata even if empty" ?